dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/utils/UrlUtils.java

```diff
@@ -134,6 +134,8 @@ public static Collection<String> allSerializations(URL url) {
                 .filter(StringUtils::isNotBlank)
                 .ifPresent(serializations::add);
         serializations.add(DefaultSerializationSelector.getDefaultRemotingSerialization());
+        System.out.println("Trying to find serializations from URL:" + url);
```

Comment:
Remove unnecessary println.

Reply:
I wll remove it later. It just used for debugging IT now :)